Sources/Epoxy/Core/ButtonRow.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  ButtonRow.swift
+//  RakuyoKit
+//
+//  Created by Rakuyo on 2024/5/17.
+//  Copyright © 2024 RakuyoKit. All rights reserved.
+//
+
+import UIKit
+
+import EpoxyCore
+import RAKCore
+
+// MARK: - ButtonRow
+
+/// Replace `UIImageView` in Epoxy component
+///
+/// If you want to extend, consider building your own view with
+/// the help of `ButtonRow.Style`, `ButtonRow.Content` and `ButtonRow.Behaviors`.
+public final class ButtonRow: UIButton {
+    /// Closure for touch down event.
+    private lazy var didTouchDown: ButtonClosure? = nil
+
+    /// Closure for tap event.
+    private lazy var didTap: ButtonClosure? = nil
+}
+
+// MARK: Action
+
+extension ButtonRow {
+    @objc
+    private func buttonDidTouchDown(_ button: UIButton) {
+        didTouchDown?(button)
+    }
+
+    @objc
+    private func buttonDidClick(_ button: UIButton) {
+        didTap?(button)
+    }
+}
+
+// MARK: StyledView
+
+extension ButtonRow: StyledView {
+    public struct Style: Hashable {
+        /// The tint color.
+        public let tintColor: UIColor?
+
+        /// The button type.
+        public let type: UIButton.ButtonType
+
+        /// The title style.
+        ///
+        /// The `color` property will be ignored.
+        /// Please use `ButtonRow.Content` to set the text color for different states.
+        public let titleStyle: TextRow.Style?
+
+        public init(
+            tintColor: UIColor? = nil,
+            type: UIButton.ButtonType = .system,
+            titleStyle: TextRow.Style? = nil
+        ) {
+            self.tintColor = tintColor
+            self.type = type
+            self.titleStyle = titleStyle
+        }
+    }
+
+    public convenience init(style: Style) {
+        self.init(type: style.type)
+
+        translatesAutoresizingMaskIntoConstraints = false
+
+        tintColor = style.tintColor
+
+        if let titleStyle = style.titleStyle {
+            titleLabel?.do {
+                $0.font = titleStyle.font
+                $0.textAlignment = titleStyle.alignment
+                $0.numberOfLines = titleStyle.numberOfLines
+                $0.lineBreakMode = titleStyle.lineBreakMode
+            }
+        }
+
+        addTarget(self, action: #selector(buttonDidTouchDown(_:)), for: .touchDown)
+        addTarget(self, action: #selector(buttonDidClick(_:)), for: .touchUpInside)
```

Comment:
These two events are relatively commonly used, so only these two events will be added for now.

Reply:
I want this encapsulation to be more pure, so I did not introduce Combine to simplify the assignment of action.